extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -57,6 +57,16 @@ public class QuartzBuildTimeConfig {
     @ConfigItem(defaultValue = "QRTZ_")
     public String tablePrefix;
 
+    /**
+     * The SQL string that selects a row in the "LOCKS" table and places a lock on the row.
+     * <p>
+     * If not set, the default value applies, for which the "{0}" is replaced during run-time with the `table-prefix`,
+     * the "{1}" with the `instance-name`.
+     * </p>
+     */
+    @ConfigItem(defaultValue = "SELECT * FROM {0}LOCKS WHERE SCHED_NAME = {1} AND LOCK_NAME = ? FOR UPDATE")
```

Comment:
I wonder if it would be safer to make this config property optional and let Quartz handle the default?

Reply:
+1

Reply:
I also considered this variant...
In the JavaDoc the usage of the {0} and {1} parameters is described, which wouldn't be useful without an example of usage. Of course, one could also expand the JavaDoc...
I would be happy with any solution...

Reply:
I would rather add an example in the Javadoc. My fear is that, at some point, Quartz changes the default value and we miss it. As we are not following closely Quartz development, that might happen.

Reply:
I think it should be okay to still document the usage of the parameters because that's how Quartz applies them.
The example SQL query can be as a javadoc as well.